.ci/openshift-ci/test-gitlab-with-pat-setup-flow.sh

```diff
@@ -25,6 +25,12 @@ export GIT_PROVIDER_URL=${GIT_PROVIDER_URL:-"https://gitlab.com"}
 export PRIVATE_REPO_SSH_URL=${PRIVATE_REPO_SSH_URL:-"git@gitlab.com:chepullreq1/private-repo.git"}
 export PRIVATE_REPO_RAW_PATH_URL=${PRIVATE_REPO_URL:-"https://gitlab.com/chepullreq1/private-repo/-/raw/main/devfile.yaml"}
 
+export PUBLIC_REPO_DOT_DEVFILE_URL=${PUBLIC_REPO_DOT_DEVFILE_URL:-"https://gitlab.com/chepullreq1/public-repo-dot-devfile.git"}
+export PRIVATE_REPO_DOT_DEVFILE_URL=${PRIVATE_REPO_DOT_DEVFILE_URL:-"https://gitlab.com/chepullreq1/private-repo-dot-devfile.git"}
```

Comment:
Variable name is a little bit confusing to me.

Did you mean something like `PRIVATE_REPO_WITH_DOT_DEFILE_URL`?